template/macros.jinja

```diff
@@ -43,22 +42,30 @@
             - salt://xxx/files/default/etc/yyy/zzz.conf.jinja
           - template: jinja
   #}
-  {%- set path_prefix = prefix|replace(':', '/') %}
-  {%- set files_switch_list = salt['pillar.get'](prefix ~ ':files_switch',
-                                           default_files_switch) %}
+  {%- set formula = 'template' %}
+  {%- set path_prefix = salt['config.get'](formula ~ ':tofs:path_prefix', formula) %}
+  {%- set files_switch_list = salt['config.get'](
+      formula ~ ':files_switch',
+      default_files_switch
+  ) %}
   {%- for grain in files_switch_list if grains.get(grain) is defined %}
     {%- for file in files %}
-      {%- set url = '- salt://' ~ '/'.join([path_prefix,
-                                            'files',
-                                            grains.get(grain),
-                                            file.lstrip('/')]) %}
+      {%- set url = '- salt://' ~ '/'.join([
+          path_prefix,
+          salt['config.get'](formula ~ ':tofs:dirs:files', 'files'),
```

Comment:
Using `config.get` to make the `'files'` directory configurable or use `'files'` by default.